### ISSUE TITLE
[SUGGESTION] Compatibility Patches

### ISSUE THREAD

**Issue by mattbratt11** [Fluffx/Nexus-Works#1]
Can Nexus be compatible with VSM, LSD, and Project Complexity in the Next Update?

**Comment by Fluffx** [comment]
Hello.

There are no plans for compatibility patches with LSD and Complexity at this time. Although, an upcoming item featured in the next Nexus update which is the Void Rune will also be featured in Complexity.

As for VSM, I can't say much about that as it generally is an addon that isn't that recommended for Complex Doom, but i'll let you know if I come to a final conclusion in the future if it should be a thing or not....

**Comment by mattbratt11** [comment]
I'm pretty sure that the April Fool Edition of Nexus is recently released like 11 Days Ago on April Fools Day.
That version contained some things from VSM, LSD, CDI, Complexity, etcetera. (Jokes not included) Is that correct?
Please let me know, as I appreciate it.

**Comment by Fluffx** [comment]
Yes, although there shouldn't be any content from LSD (unless it's hem-custom) and Complexity, otherwise what you're saying is true. Keep in mind that the april fools version is not an official nexus update but made for the April month exclusively.

**Comment by mattbratt11** [comment]
For the next update of Nexus, can you at least add a new "April Fool Mode" option so we can enjoy the April Fool Edition year-round (like anytime we want to)?

Also, can Nexus be Compatible with Paradox's CF Revised Addon as an additional patch?
